recipes/drogon/all/patches/1.7.5-0005-remove-msvc-check.patch

```diff
@@ -0,0 +1,92 @@
+diff --git a/lib/inc/drogon/utils/any.h b/lib/inc/drogon/utils/any.h
+index 63abd2e..8ac74d8 100644
+--- a/lib/inc/drogon/utils/any.h
++++ b/lib/inc/drogon/utils/any.h
+@@ -13,7 +13,7 @@
+  */
+ 
+ #pragma once
+-#if __cplusplus >= 201703L || (defined _MSC_VER && _MSC_VER > 1900)
++#if __cplusplus >= 201703L
```

Comment:
If I'm not mistaken, Visual C++ defines the `__cplusplus` macro to be `199711L` by default, unless a flag is passed to the compiler (docs: https://learn.microsoft.com/en-us/cpp/build/reference/zc-cplusplus?view=msvc-170) 
Hence the `||` checks, checking against the value of `_MSC_VER` - I can se how this can be misleading: modern Visual C++ defaults to C++ 14, but these are features of C++17 that seem available even when in C++14 (default) mode, correct?


So this may be causing the "boost" implementation to be used unconditionally with Visual C++, even when in C++17 or C++20 modes - is this expected?